checker/tests/src/tests/OIGJTest.java

```diff
@@ -21,8 +22,8 @@ public OIGJTest(File testFile) {
     }
 
     @Parameters
-    public static Collection<Object[]> data() {
-        return filter(testFiles("oigj", "all-systems"));
+    public static Collection<Object[]> getTestFiles() {
+        return filter(TestUtilities.findNestedJavaTestFiles("oigj", "all-systems"));
     }
 
     // Duplicate from JavariTest.
```

Comment:
Is it worth moving the filter functionality into TestUtilities?
No, actually we should remove that code here and skip-test the files that are filtered.